plugins/kernel_v2/config/vm.rb

```diff
@@ -123,6 +126,28 @@ def merge(other)
             end
           end
 
+          # Merge defined disks
+          other_disks = other.instance_variable_get(:@disks)
+          new_disks   = []
+          @disks.each do |p|
+            other_p = other_disks.find { |o| p.id == o.id }
```

Comment:
it would be nice if we could do merging by `id` or by `name` since from the user point of view using `name` to identify disks would be much easier.